src/java.base/share/classes/sun/nio/ch/ChannelInputStream.java

```diff
@@ -265,11 +265,43 @@ public long transferTo(OutputStream out) throws IOException {
             return transfer(rbc, fc);
         }
 
+        // ReableByteChannel -> WritableByteChannel
```

Comment:
Typo, should be "ReadbleByteChannel"

Reply:
> Typo, should be "ReadbleByteChannel"

[sic] / pun intended 😉? Thank you, good catch! I fixed it in 0a7e020928f010c42477df5f04c69b915182bbac.